src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -357,6 +353,11 @@ public override void _Ready()
 
         mapDrawer.OnSelectedPatchChanged = (drawer) => { UpdateShownPatchDetails(); };
 
+        atpProductionBar.Type = "ATP";
+        atpConsumptionBar.Type = "ATP";
+        atpProductionBar.Size = new Vector2(318, 30);
```

Comment:
The size shouldn't be hardcoded, it should either automatically detect the available size, or it needs to be set in the GUI scene.